### PR TITLE
Tmedia 482 footer tap targets

### PR DESCRIPTION
**Pull request by JackHowa**
## Description
Increase margin between links, increase tappability size to match mocks

Also used the mocks to find the actual target size of the links to match mocks https://app.zeplin.io/project/603fa53e2626ed1592e7c0e6/screen/603fa5c34a388fa3d696150e
Screen Shot 2021-12-21 at 15 43 50

increased text size from 14px -> 16px and used rem. also increased line height to match mocks of surrounding elements per mocks

## Jira Ticket
- [TMEDIA-482](https://arcpublishing.atlassian.net/browse/TMEDIA-482)

## Acceptance Criteria
All footer links pass lighthouse tapability test 

## Test Steps

1. Checkout this branch `git checkout TMEDIA-482-footer-tap-targets`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/footer-block`
3. Go to http://localhost/pf/all-blocks/?_website=the-sun
4. Run lighthouse in the dev tools 
5. You should see the tappability rating for footers pass 

### Before

In the cdn lighthouse test: https://lighthouse-dot-webdotdevsite.appspot.com//lh/html?url=https%3A%2F%2Fcorecomponents-the-gazette-prod.cdn.arcpublishing.com%2Fall-blocks%2F

locally: 
Screen Shot 2021-12-21 at 15 27 20

### After
Screen Shot 2021-12-21 at 15 26 03

## Dependencies or Side Effects
- The footer items are as intended by mocks now 

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
- [x] Confirmed all the test steps a reviewer will follow above are working.
- [x] Confirmed there are no linter errors. Please run `npm run lint` to check for errors. Often, `npm run lint:fix` will fix those errors and warnings.
- [x] Ran this code locally and checked that there are not any unintended side effects. For example, that a CSS selector is scoped only to a particular block.
- [ ] Confirmed this PR has reasonable code coverage. You can run `npm run test:coverage` to see your progress.
  - [ ] Confirmed this PR has unit test files
  - [ ] Ran `npm run test`, made sure all tests are passing
  - [ ] If the amount of work to write unit tests for this change are excessive,
please explain why (so that we can fix it whenever it gets refactored). -> not applicable
- [x] Confirmed relevant documentation has been updated/added. -> not applicable

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._

- [ ] Linting code actions have passed.
- [ ] Ran the code locally based on the test instructions.
  - [ ] I don’t think this is needed to be tested locally. For example, a padding style change (storybook?) or a logic change (write a test).
- [ ] I am a member of the engine theme team so that I can approve and merge this. If you're not on the team, you won't have access to approve and merge this pr.
- [ ] Looked to see that the new or changed code has code coverage, specifically. We want the global code coverage to keep on going up with targeted testing.

### CODE REVIEW

File: blocks/footer-block/features/footer/footer.scss [hunk 1]
```diff
@@ -139,17 +153,10 @@ footer {
...
-    }
-
-    .footer-item {
-      margin: map-get($spacers, 'xs') 0;
-
-      > a {
-        color: $ui-medium-gray;
-        text-decoration: none;
-      }
+      font-size: calculateRem(16px);
```
Comment (matthewroach) [inline blocks/footer-block/features/footer/footer.scss]: We are setting explicit font-sizes and line-height on multiple times in this file to all be the same values - why don't we move up the declarations to the top level `footer {` and let CSS do it's thing, and only change as needed?

For the `.footer-header` you'll likely need to do the follow as they are heading elements
```
      font-size: inherit;
      line-height: inherit;
```

**Review by matthewroach — CHANGES_REQUESTED** [review]
We should clean up the font size and line height rules a little to be cleaner

File: blocks/footer-block/features/footer/footer.scss (hunk 1 above)
Comment (JackHowa) [reply blocks/footer-block/features/footer/footer.scss:156]: cool updated ✅

**Comment by JackHowa** [comment]
weird not pushing ... one sec

File: blocks/footer-block/features/footer/footer.scss (hunk 1 above)
Comment (JackHowa) [reply blocks/footer-block/features/footer/footer.scss]: updated

**Review by matthewroach — APPROVED** [review]

**Pull request merged**